templates/server/fastcgi_params.erb

```diff
@@ -5,12 +5,13 @@ fastcgi_param  REQUEST_METHOD     $request_method;
 fastcgi_param  CONTENT_TYPE       $content_type;
 fastcgi_param  CONTENT_LENGTH     $content_length;
 
-fastcgi_param  SCRIPT_FILENAME    $request_filename;
```

Comment:
Upstream has two files, fastcgi.conf and fastcgi_params. As written the PR syncs with fastcgi_params which doesn't set `SCRIPT_FILENAME`. fastcgi.conf is more modern and does set it (that's the only difference between them).

I suggest syncing to fastcgi.conf, and renaming this template to `fastcgi.conf.erb`... that's likely to have the least impact on users while also being explicit about which upstream fastcgi config file it's trying to replicate.

Some background here if you like: https://blog.martinfjordvald.com/2013/04/nginx-config-history-fastcgi_params-versus-fastcgi-conf/